src/ducks/zkpr.ts

```diff
@@ -137,7 +115,10 @@ export const createZKPRIdentity =
 
 export async function maybeSetZKPRIdentity(idCommitment: string) {
   let id: Identity | null = null;
-  const data = await checkPath(idCommitment);
+  const data = await checkPath(idCommitment).catch(err => {
```

Comment:
we should use `findProof` from `utils/merkle` instead.

The util method now requires a group id, but can actually be adjusted to make group id an optional param, as zkitter api can query and return the first matching proof.

like this: https://api.zkitter.com/v1/proofs/2533fc40284a52cb44b7e7fdd9ac43ef4c5e9160d8150c3b12015414f8ae3295